### ISSUE TITLE
Problem with ++ operator

### ISSUE THREAD

**Issue by nulano** [koofr/javatoscala#4]
```
int i = 5;
System.out.println(i++);
```

converts to

```
var i = 5
println(i += 1)
```

which doesn't compile

**Comment by nightscape** [comment]
The newest version of Scalagen translates this to
```scala
var i: Int = 5
println({ i += 1; i - 1 })
```
so I think this issue can be necro-closed :wink:

**Comment by bancek** [comment]
Fixed with #9.
